internal/status/status.go

```diff
@@ -0,0 +1,8 @@
+package status
+
+//Service is an interface responsible to add/update user status to idler.
+type Service interface {
+	CheckUserStatus(user string) (bool, error)
```

Comment:
https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

Reply:
@pradeepitm12 what I meant by the comment above is to make use of [golang enum](https://blog.learngoprogramming.com/golang-const-type-enums-iota-bc4befd096d3?gi=748931a8c6a3) so that UserStatus(user string) returns `( Status, error)`